### ISSUE TITLE
General component accessibility

### ISSUE THREAD

**Issue by marcysutton** [lumapps/lumX#69]
Every component I have checked is sorely lacking in accessibility support. For your framework to be usable, it must support keyboards and assistive technologies such as screen readers. Is this on your roadmap?

**Comment by malexandre** [comment]
The roadmap is now available here : https://github.com/lumapps/lumX/blob/master/ROADMAP.md

**Comment by marcysutton** [comment]
Is accessibility going to be worked into every component from now on?

**Comment by dmackerman** [comment]
Not being able to select options from a dropdown/select with your keyboard is kind of a deal breaker for a lot of components. Would love to help out where possible!

**Comment by malexandre** [comment]
I've started a first component with accessibility in mind (the slider). We'll push it to test the accessibility and think it in a way to be applied for all the components. Once there, we'll communicate to let anyone know how to do it and help if they want.
I just watch the talk from @marcysutton at ng-europe about that.

**Comment by marcysutton** [comment]
@malexandre awesome!!

**Comment by jusopi** [comment]
keyboard support is a must for the dropdown/select components.... ui-bootstrap lacks the mouse-click support for their typeahead but supports keyboard navigation/selection.  Maybe a solution can be gleaned from their code?
